services/openshift/scripts/openshift_provision

```diff
@@ -120,5 +120,9 @@ if [ ! -f ${ORIGIN_DIR}/configured.user ]; then
   echo "[INFO] Adding required roles to openshift-dev and admin user ..."
   oadm policy add-role-to-user basic-user openshift-dev --config=${OPENSHIFT_DIR}/admin.kubeconfig
   oadm policy add-cluster-role-to-user cluster-admin admin --config=${OPENSHIFT_DIR}/admin.kubeconfig
+  oc login https://127.0.0.1:8443 -u openshift-dev -p devel \
+        --certificate-authority=${OPENSHIFT_DIR}/ca.crt &>/dev/null
+  oc new-project test --display-name="OpenShift 3 Sample" \
```

Comment:
> I don't think you can create a project called default here - there's already a default project existing

True, true. A very unfortunate choice of the OpenShift team imo. Just 'openshift' does not fly either :-( Anyways, what's about 'localhost' or just 'local'? There are two aspects to this name. First it should be understandable as to the user that this is his/her project workspace. From this perspective 'openshift-dev' might make more sense. On the other hand the project will also be part of the default route and we should aim for "nice" and descriptive URLs. Using 'local' would give us things like 'mynode-local.rhel-cdk.10.1.2.2.xip.io'.